### ISSUE TITLE
Sample 1 has an invalid maximum Phred Quality Scores of 496

### ISSUE THREAD

**Issue by otaviolovison** [benjjneb/dada2#1607]
Hello!

I am getting this error when running dada. I've already tried > errF <- learnErrors(filtFs, multithread=TRUE) and > dadaFs <- dada(derepFs, err=errF, pool=TRUE, multithread = TRUE) and dadaFs <- dada(derepFs, err=ddF[[1]]$err_out, pool=TRUE, multithread = TRUE).

Any idea on how can I solve this?

Feel free to ask for more details. Thank you in advance!

**Comment by benjjneb** [comment]
Not sure how that is even possible.
Is this fastq file ASCII encoded as expected?
Does it look normal, e.g. when you `less my_sample.fastq`?

**Comment by otaviolovison** [comment]
I agree with you. Something is going very wrong here.. but the file is encoded as expected.

**Comment by benjjneb** [comment]
When you say it is "encoded as expected" how are you determining that? Just the output of `less` looking normal?

One thing to try might be to open up a fastq file, and then save it to a new file making sure it is saved in ASCII format. Then try on the new file to see if this error crops up again.

There also could be an issue with carriage returns if this file was saved in Windows format at some point (although how that would get a Q score of 496 I still don't understand).

**Comment by otaviolovison** [comment]
Hello!

Sorry for the late reply. I had to travel for a symposium and just checked now the results.
I don't know if this is the problem, but here what happened:
I was trying to use 'Quarto' for the first time. I noticed that using Quarto, the computation became really slow. And then, I had this problem. 
Now, running the analysis without Quarto, everything went well. 
Any reports like that? What could be happening?
